docs/Using-Fleet/REST-API.md

```diff
@@ -3122,6 +3122,46 @@ Retrieves the disk encryption key for a host.
 }
 ```
 
+### Get configuration profiles assigned to a Host 
+
+Requires Fleet's MDM properly [enabled and configured](./Mobile-device-management.md).
+
+Retrieves a list of the configuration profiles assigned to the Host.
+
+`GET /api/v1/fleet/mdm/hosts/:id/profiles`
+
+#### Parameters
+
+| Name | Type    | In   | Description                      |
+| ---- | ------- | ---- | -------------------------------- |
+| id   | integer | path | **Required** The id of the host  |
```

Comment:
I know it's a bit inconsistent right now, but would add punctuation for readability. 
```suggestion
| id   | integer | path | **Required**. The ID of the host  |
```
(Same for formatting of "id"/"ID" — inconsistent on this page, but "ID" is preferable in the description.)